github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -169,12 +169,13 @@ presubmits:
     annotations:
       fork-per-release: "true"
       k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
-    always_run: false
-    optional: true
+    always_run: true
+    optional: false
+    skip_report: false
     decorate: true
     decoration_config:
-      timeout: 3h
-      grace_period: 20m
+      timeout: 4h
+      grace_period: 30m
```

Comment:
Why this change is required?
We have only a few SRIOV tests, why do we need 4h? (even 3 sounds too much).

Reply:
It used to be 5 hours and I reduced it to 3 hours,
in one of the iterations I saw that kubevirt may fail to deploy at frist or take long time to finish, 
then tests are cuts off and the job is finished due to timeout.